pfcpiface/parse-sdf.go

```diff
@@ -21,7 +22,7 @@ var errBadFilterDesc = errors.New("unsupported Filter Description format")
 
 type endpoint struct {
 	IPNet *net.IPNet
-	Port  uint16
+	ports portFilter
```

Comment:
What about:

```suggestion
	port portRange
```

Reply:
My idea was keeping the name generic, as not all ternary matches are ranges (e.g. mask low bits). But since PFCP does not allow this, `PortRange` makes more sense?

Reply:
yep, `PortRange` makes sense